js/tests/unit/scrollspy.js

```diff
@@ -231,48 +235,40 @@ $(function () {
 
   QUnit.test('should correctly select navigation element on backward scrolling when each target section height is 100%', function (assert) {
     assert.expect(5)
+    var done = assert.async()
+
+    var navLinks = ''
+    var contentDivs = ''
+    var i = 0
+
+    while (++i < 6) {
+      navLinks += '<li id="li-100-' + i + '"><a href="#div-100-' + i + '">div ' + i + '</a></li>'
+      contentDivs += '<div id="div-100-' + i + '" style="position: relative; height: 100%; padding: 0; margin: 0">div ' + i + '</div>'
+    }
+
     var navbarHtml =
         '<nav class="navbar">'
       + '<ul class="nav">'
-      + '<li id="li-100-1"><a href="#div-100-1">div 1</a></li>'
```

Comment:
It's not a problem I can change it back to my previous version. The only thing it would be great if you would have one opinion with your other team members, either it looks like a heavy trolling asking me to change it back and forth :)